ortac-qcheck-stm.opam

```diff
@@ -32,10 +32,8 @@ depends: [
   "qcheck-core" {with-test}
   "qcheck-stm" {with-test}
   "qcheck-multicoretests-util" {with-test}
-  "ortac-core"
+  "ortac-core" {= "0.2.0"}
```

Comment:
Both dune and opam support the `version` variable.
A dune example here: https://github.com/ocaml-multicore/multicoretests/blob/main/dune-project#L35
A (generated) opam example here: https://github.com/ocaml-multicore/multicoretests/blob/main/qcheck-stm.opam#L27
Using it has the advantage of needing to go around and updating many version numbers on each release :slightly_smiling_face:

Reply:
Nice!